.gitignore

```diff
@@ -205,3 +205,5 @@ pg_data
 .ruby-version
 mysql2psql.yml
 *.pem
+
+Gemfile.lock
```

Comment:
I forgot to mention, you may also need to do version control on `Gemfile.lock`. So please remove line 208 and line 209.